client/v2/client.go

```diff
@@ -119,6 +123,49 @@ func NewHTTPClient(conf HTTPConfig) (Client, error) {
 	}, nil
 }
 
+// Pings cluster
+func (c *client) Ping(timeout time.Duration) (time.Duration, string, error) {
```

Comment:
`golint` won't like this comment. Can you put something more descriptive, beginning with `Ping...`. Possibly take your description on the interface and put it here, with a short description on the interface.